setupCOPR.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 
+if [ "$COPR_API" == "" ]; then
+    echo "ERROR: COPR_API environment variable undefined"
```

Comment:
Additional line?
`echo "NOTE: Travis won't inject Secure variables in Pull Request"`

Reply:
Fixed.